cli/command/environment_test.go

```diff
@@ -56,12 +55,9 @@ func TestCreateEnvironmentInputErrors(t *testing.T) {
 
 	contexts := map[string]*cli.Context{
 		"Missing NAME arg": testutils.NewTestContext(t, nil, nil),
-		"Negative MinScale": testutils.NewTestContext(t,
+		"Negative Scale": testutils.NewTestContext(t,
 			[]string{"env_name"},
-			map[string]interface{}{"min-scale": "-1"}),
-		"Negative MaxScale": testutils.NewTestContext(t,
-			[]string{"env_name"},
-			map[string]interface{}{"max-scale": "-1"}),
+			map[string]interface{}{"scale": "-1"}),
```

Comment:
I think it would be useful to add a non-integer case, e.g. `{"scale": "one"}`

Reply:
Not sure I agree here. This wouldn't be testing our code; rather that a IntFlag will throw an error if an invalid int is passed in.

Reply:
oh shoot, sorry I got test mixed up with `EnvironmentScale` where that's an arg, not a flag.